api/v1alpha3/conversion.go

```diff
@@ -201,3 +201,58 @@ func Convert_Slice_v1alpha3_Network_To_Slice_v1beta1_Network(in *[]Network, out
 	}
 	return nil
 }
+
+func Convert_v1alpha3_SubnetFilter_To_v1beta1_SubnetFilter(in *SubnetFilter, out *v1beta1.SubnetFilter, s conversion.Scope) error {
+	out.Name = in.Name
+	out.Description = in.Description
+	if in.ProjectID != "" {
+		out.ProjectID = in.ProjectID
+	} else {
+		out.ProjectID = in.TenantID
+	}
```

Comment:
nit: we could write this without the else

```suggestion
	out.ProjectID = in.TenantID
	if in.ProjectID != "" {
		out.ProjectID = in.ProjectID
	}
```

Reply:
This is a toss-up. I probably wouldn't bother updating this one.